ui/src/actions/sampleGrid.js

```diff
@@ -50,33 +47,6 @@ export function addSamplesToList(samplesData) {
   };
 }
 
-export function setSampleOrderAction(order) {
-  return { type: 'SET_SAMPLE_ORDER', order };
-}
-
-/**
- * @deprecated New Sample Grid does not allowed sample reordering
```

Comment:
I've removed this deprecated thunk and the corresponding action and reducer `case` block.